dhis2/dashboard-app
# fix: local offline handling [DHIS2-12937]

Fixes the error in [DHIS2-12937](https://dhis2.atlassian.net/browse/DHIS2-12937)

Seems to depend on this PR in the analytics library: https://github.com/dhis2/analytics/pull/1443

### Description

- Switches from `useOnlineStatus` to `useDhis2ConnectionStatus` to detect connection to the DHIS2 server instead of the network status

### Testing instructions

See the [comment on LIBS-315](https://dhis2.atlassian.net/browse/LIBS-315?focusedCommentId=181353)



[DHIS2-12937]: https://dhis2.atlassian.net/browse/DHIS2-12937?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
